README.md

```diff
@@ -30,7 +30,7 @@ For example, GuessIt can do the following:
         "type": "episode"
     }
 
-More information are available at [guessit.io](http://guessit.io/).
+More information is available at [guessit-io.github.io/guessit](https://guessit-io.github.io/guessit).
```

Comment:
This link is also broken...  I think this might be the best source for documentation: https://guessit.readthedocs.io/en/latest/ ? Or the wayback machine: https://web.archive.org/web/20210619145049/https://doc.guessit.io/

Reply:
Once the gh-pages branch is pushed without the `docs/CNAME` file that's being removed here, the link _should_ work as expected. I used that particular link because it's the same one as in the sidebar of github, if guessit.readthedocs.io is preferred I can change to that.